<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicPlayer</class>
 <widget class="QMainWindow" name="MusicPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>180</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>180</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>180</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>播放器</string>
  </property>
  <property name="windowIcon">
   <iconset resource="MusicPlayer.qrc">
    <normaloff>:/res/ico/ico2.ico</normaloff>:/res/ico/ico2.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-image: url(:/res/img/bg.png);
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="fileButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>65</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>11</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-image: url();
	background-color:rgba(200, 200, 200, 80);
	border-color:rgba(100, 100, 100, 80);
	color:white;
}


</string>
    </property>
    <property name="text">
     <string>打开</string>
    </property>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>65</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();
background-color:rgba(200, 200, 200, 80);
border-color:rgba(100, 100, 100, 80);
color:white;
</string>
    </property>
    <property name="text">
     <string>播放</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="musicName">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url();
background-color:rgba(100, 100, 100, 150);
color:white;
font-size:12px;
border:none;</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:12px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>65</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();
background-color:rgba(200, 200, 200, 80);
border-color:rgba(100, 100, 100, 80);
color:white;
</string>
    </property>
    <property name="text">
     <string>下一首</string>
    </property>
   </widget>
   <widget class="QLabel" name="nowTime">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:none;
color:white;</string>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="totalTime">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>140</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:none;
color:white;</string>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="slider">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>155</y>
      <width>280</width>
      <height>14</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider:horizontal#slider {
	background: none;
	border: none;
}

QSlider::groove:horizontal#slider {
	background: none;
	border-color:none;
}

QSlider::handle:horizontal#slider {
	background-color:rgba(255, 255, 255, 200);
	width: 10px;
	margin-top:2px;
	margin-bottom:2px;
	border: none;
	border-radius: 5px;
}

QSlider::handle:horizontal:hover#slider {
	background-color:rgba(255, 255, 255, 255);
	width: 10px;
	margin-top:2px;
	margin-bottom:2px;
	border: none;
	border-radius: 5px;
}

QSlider::sub-page:horizontal#slider {
	background-color:rgba(102, 204, 255, 150);
	border: none;
	border-radius: 2px;
	margin-top:5px;
	margin-bottom:5px;
}

QSlider::add-page:horizontal#slider {
	background-color:rgba(70, 70, 70, 150);
	border: none;
	border-radius: 2px;
	margin-top:5px;
	margin-bottom:5px;
}</string>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="tracking">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="volumeSlider">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>14</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider:vertical#volumeSlider {
	background: none;
	border: none;
}

QSlider::groove:vertical#volumeSlider {
	background: none;
	border-color:none;
}

QSlider::handle:vertical#volumeSlider {
	background-color:rgba(255, 255, 255, 200);
	height: 10px;
	margin-left: 2px;
	margin-right: 2px;
	border: none;
	border-radius: 5px;
}

QSlider::handle:vertical:hover#volumeSlider {
	background-color:rgba(255, 255, 255, 255);
	height: 10px;
	margin-left: 2px;
	margin-right: 2px;
	border: none;
	border-radius: 5px;
}

QSlider::add-page:vertical#volumeSlider {
	background-color:rgba(102, 204, 255, 150);
	border: none;
	border-radius: 2px;
	margin-left:5px;
	margin-right:5px;
}

QSlider::sub-page:vertical#volumeSlider {
	background-color:rgba(70, 70, 70, 150);
	border: none;
	border-radius: 2px;
	margin-left:5px;
	margin-right:5px;
}</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>125</y>
      <width>30</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: none;
color:white;</string>
    </property>
    <property name="text">
     <string>音量</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextBrowser" name="lyrics">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url();
background-color:rgba(100, 100, 100, 150);
color:white;
font-size:12px;
border:none;</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:12px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:15px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MusicPlayer.qrc"/>
 </resources>
 <connections/>
</ui>
